app/components/app/app.component.ts

```diff
@@ -104,4 +121,14 @@ export class AppComponent {
       }
     });
   }
+
+  translateConfig() {
+    var userLang = navigator.language.split('-')[0];
```

Comment:
I don't know if I'm missing something, but where is navigator declared?
And how does this work, is the systems language set as default?